lsp/nls/src/requests/symbols.rs

```diff
@@ -24,12 +28,15 @@ pub fn handle_document_symbols(
                     let range = item
                         .pos
                         .try_to_range()
-                        .or_else(|| Some((file_id.clone(), (0usize..0usize))))
+                        .or_else(|| Some((file_id, (0usize..0usize))))
                         .map(|(file_id, range)| {
                             codespan_lsp::byte_span_to_range(server.cache.files(), file_id, range)
                                 .unwrap()
                         })
                         .unwrap();
+
+                    // `deprecated` is a required field but causes a warning although we are not using it
+                    #[allow(deprecated)]
```

Comment:
Yeah, I really don't understand how you are supposed to handle those deprecated warning for struct fields. It's still in the type definition, it's not like we have the choice...